packages/core/src/styles/typography/_typography.scss

```diff
@@ -373,9 +373,13 @@
 
 [cds-text~='link'] {
   color: $cds-token-color-action-600 !important;
-  text-decoration: underline !important;
+  text-decoration: none !important;
```

Comment:
This is something that accessibility rightly should and probably will ding us on. Having a link indicated by color alone is not accessible. I get that this is aligning core with what we have in clarity CSS today.

Reply:
Ah good point, I thought it was a bug since clr-ui does not have the underline. I can open a new PR to fix it on master and then backport it to v4 for a later release.

Reply:
Yeah. Visually it would be a breaking change. So I don't think there's a path to backport it? I don't know how we balance the whole breaking change versus accessibility question.

Reply:
I have to rebase anyways so I can pull this change out and follow up with a fix on master